### PR TITLE
remove unnecessary range check

### PR DESCRIPTION
**Pull request by ishmum123**

### CODE REVIEW

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnectorTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    private final Teradata14LogsConnector connector = new Teradata14LogsConnector();
+
+    @Test
+    public void testConnector() throws Exception {
+        testConnectorDefaults(connector);
+    }
+
+    @Test
+    public void testFailsForInvalidQueryLogTimespan() throws IOException {
```
[the remark this answers is not in the document]
Comment (ishmum123) [reply dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnectorTest.java]: Implemented for `AbstractJdbcConnector` as `LogsConnector` has implementations which do not comply (e.g. - `BigQueryLogsConnector`)

**Review by abogdanov93 — APPROVED** [review]

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/AbstractJdbcConnectorTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Google LLC
```
Comment (paolomorandini) [inline dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/AbstractJdbcConnectorTest.java:2]: Note: we should update the template to `2023`. Not required in this PR.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java [hunk 3]
```diff
@@ -161,21 +161,14 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
         out.add(new DumpMetadataTask(arguments, FORMAT_NAME));
         out.add(new FormatTask(FORMAT_NAME));
 
-        String queryFormat = newQueryFormat(arguments);
-
-        final int daysToExport = arguments.getQueryLogDays(7);
-        if (daysToExport <= 0)
-            throw new MetadataDumperUsageException("At least one day of query logs should be exported; you specified: " + daysToExport);
-        LOG.info("Log entries within the last {} days will be exported in increments of 1 hour.", daysToExport);
```
Comment (paolomorandini) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java:169]: Is this message going away on purpose?

**Review by paolomorandini — APPROVED** [review]

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/AbstractJdbcConnectorTest.java (hunk 2 above)
Comment (ishmum123) [reply dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/AbstractJdbcConnectorTest.java:2]: Noted, Thanks!

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java (hunk 3 above)
Comment (ishmum123) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java:169]: It's already logged in [here](https://github.com/google/dwh-migration-tools/blob/858e7988f048024eeb9593adaf5d84a395c2c1e0/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java#L97)

**Pull request merged**